src/webview/utils/messageListener.ts

```
@@ -0,0 +1,38 @@
+import { isDev } from '.';
+import { IWebviewMessage, WebviewMessageType } from '../../utils/extType';
+import { TokenHandler, WebviewMessageHandler } from './receiveStrategy';
+
+class MessageListener {
+  handlers: Map<WebviewMessageType, WebviewMessageHandler> = new Map();
+
+  static instance: MessageListener | null;
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)